src/index.js

```diff
@@ -0,0 +1,5 @@
+import types from './types'
+import infer from './infer'
+import Schema from './schema'
+
+module.exports = { infer, types, Schema }
```

Comment:
No, it is not as simple as it looks like. Need to refactor tests completely, and in type guesser create additional methods to have a possibility to cast by type